### PR TITLE
ZackM - Adding Example for Flex

### PR DESCRIPTION
**Pull request by thezackm**

### CODE REVIEW

File: .gitignore [hunk 1]
```diff
@@ -25,3 +25,6 @@ yarn-error.log*
 # build data
 public/data
 src/data.json
+
+# quickstart generate data shell file
```
Comment (kidk) [inline .gitignore:29]: Why did you add this to gitignore? You're confusing me 😃

Comment (thezackm) [reply .gitignore:29]: it was marked as `modified` as soon as I forked/cloned; I've only seen that happen on files that constantly have non-breaking changes and should be ignored. I can revoke that if you'd like.

Comment (kidk) [reply .gitignore:29]: You can remove this. For some reason the permissions got changed on the file from 755 to 644.

Comment (thezackm) [reply .gitignore]: reverted with commit bc609b6
